docs/product-installation-guide/src/main/asciidoc/clustering-headless-controller-proc.adoc

```diff
@@ -1,12 +1,13 @@
 [id='clustering-headless-controller-proc']
 = Clustering with the headless {PRODUCT_SHORT} controller
 
+////
 You can configure {KIE_SERVER} to run in managed or unmanaged mode. If {KIE_SERVER} is unmanaged, you must manually create and maintain containers. If {KIE_SERVER} is managed, the {PRODUCT_SHORT} controller manages the {KIE_SERVER} configuration and you interact with the controller to create and maintain containers.
```

Comment:
Shouldn't this be rather deleted than just commented out?

Reply:
I commented out this para rather than deleting because this particular topic is being used in other docs as well.

Reply:
Ok then.